lua-lace-dev-1.rockspec

```diff
@@ -0,0 +1,28 @@
+package = "lua-lace"
+version = "dev-1"
+source = {
+   url = "git+https://git@github.com/lunarmodules/lua-lace.git"
```

Comment:
If you're going to use http you don't need the username
```suggestion
   url = "git+https://github.com/lunarmodules/lua-lace.git"
```

Reply:
We might want to go a step further, and remove the `git+` prefix too. The official URL that Github provides for unauthenticated pulls is `https://github.com/lunarmodules/lua-lace.git`

Reply:
Oops, yes that should go too. I'll amend...